File: financial-domain-starter/src/main/java/com/nixsolutions/financial/security/exception/TokenExpiredException.java
package com.nixsolutions.financial.security.exception;

public class TokenExpiredException extends RuntimeException
{
  public TokenExpiredException()
  {
  }

  public TokenExpiredException(String s)
  {
    super(s);
  }

  public TokenExpiredException(String s, Throwable throwable)
  {
    super(s, throwable);
  }

  public TokenExpiredException(Throwable throwable)
  {
    super(throwable);
  }
}

File: financial-domain-starter/src/main/kotlin/com/nixsolutions/financial/security/exception/TokenExpiredException.kt
package com.nixsolutions.financial.security.exception

class TokenExpiredException : RuntimeException {

  @JvmOverloads constructor(s: String = "Token expired") : super(s) {}

  constructor(s: String, throwable: Throwable) : super(s, throwable) {}

  constructor(throwable: Throwable) : super(throwable) {}
}
